src/main/java/de/fraunhofer/aisec/analysis/utils/Utils.java

```diff
@@ -267,6 +267,23 @@ public static String toNonQualifiedName(@NonNull String typeStr) {
 		return typeStr;
 	}
 
+	/**
+	 * Counterpart to toNonQualifiedName().
+	 *
+	 * @param fqn
+	 * @return
+	 */
+	@NonNull
+	public static String getScope(@NonNull String fqn) {
+		int posDot = fqn.lastIndexOf('.');
+		int posColon = fqn.indexOf(':');
```

Comment:
Shouldn't this be `fqn.lastIndexOf(':') - 1`?

Reply:
Almost, yes. It should be `lastIndexOf("::")`. I added the fix to this PR.